crypto/ube/ube_test.cc

```diff
@@ -20,20 +53,41 @@ TEST(Ube, BasicTests) {
   ASSERT_EQ(current_generation_number, generation_number);
 }
 
-TEST(Ube, MockedMethodTests) {
+TEST_F(ubeTest, MockedMethodTestsSnapsafe) {
+
+  allowMockedUbe();
+
   uint64_t generation_number = 0;
   uint64_t cached_generation_number = 0;
-  if (CRYPTO_get_ube_generation_number(&generation_number) == 0) {
-    // In this case, UBE detection is disabled, so just return
-    // successfully. The short-circuit feature means we can't mock detection
-    // methods.
-    return;
-  }
 
-  // The fork generation number is initially 1. Use 10 because it's larger...
-  // Configuring specific values must change later on, since we might have tests
-  // running concurrently.
-  set_fork_generation_number_FOR_TESTING(10);
+  // First test incrementing the fork generation number. Pick a starting point
+  // and get initial UBE generation number
+  set_fork_generation_number_FOR_TESTING(5);
+  ASSERT_TRUE(CRYPTO_get_ube_generation_number(&generation_number));
```

Comment:
Should this be `set_snapsafe_generation_number_FOR_TESTING` and talk about testing snapsafe generation number in the comment?